src/main/java/com/codegym/jrugotom5/dto/AdvertInfoForCreatorDto.java

```diff
@@ -0,0 +1,14 @@
+package com.codegym.jrugotom5.dto;
+
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@Data
+public class AdvertInfoForCreatorDto {
```

Comment:
What does mean "AdvertInfoForCreatorDto"?
Why can't you use AdvertBasicInfoDTO?

Reply:
> What does mean "AdvertInfoForCreatorDto"? Why can't you use AdvertBasicInfoDTO?

i think we need to return full advert info for this api, so better to rename this dto to AdvertFullInfoDTO

Reply:
> > What does mean "AdvertInfoForCreatorDto"? Why can't you use AdvertBasicInfoDTO?
> 
> i think we need to return full advert info for this api, so better to rename this dto to AdvertFullInfoDTO

I think I might need isActive field that is not presenting in AdvertFullInfoDTO